server/src/ItemList.tsx.ts

```diff
@@ -3318,8 +3138,7 @@ const uniqueItems = {
       "+130% Enhanced Damage",
       "Ignore Target's Defense",
       "Prevent Monster Heal",
-      "+8 to LIght Radius",
-      "found in v1.11 or later"
+      "+8 to LIght Radius"
```

Comment:
just saw this typoe here - maybe you can fix it ;)

```suggestion
      "+8 to Light Radius"
```